test/get_previous_releases.py

```
@@ -327,7 +320,12 @@ def main(args) -> int:
                         help='download release binary.')
     parser.add_argument('-t', '--target-dir', action='store',
                         help='target directory.', default='releases')
-    parser.add_argument('tags', nargs='+',
-                        help="release tags. e.g.: v19.1.0 v19.0.0-rc.9")
+    parser.add_argument('tags', nargs='*', default=set(
+                            [v['tag'] for v in SHA256_SUMS.values()]
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)